### PR TITLE
Add flag to replay for rr onprocess pid

### PR DESCRIPTION
**Pull request by jcpowermac**
Add `-p` or `rr-onprocess-pid` to replay
command to pass `-p` or `--onprocess` pid
to `rr`.

### CODE REVIEW

**Comment by jcpowermac** [comment]
I was unsure about submitting this PR. If there is an alternative way to do this please let me know.

When using rr with openshift installer I couldn't find a way to debug the terraform providers which is forked from terraform and terraform is forked from the installer.

File: cmd/dlv/cmds/commands.go [hunk 1]
```diff
@@ -368,6 +370,10 @@ https://github.com/mozilla/rr
 				os.Exit(execute(0, []string{}, conf, args[0], debugger.ExecutingOther, args, buildFlags))
 			},
 		}
+
+		replayCommand.Flags().IntVarP(&rrOnProcessPid, "rr-onprocess-pid", "p", 0,
```
Comment (aarzilli) [inline cmd/dlv/cmds/commands.go:374]: I think we should just rename the long version of this to `--onprocess`.

**Review by aarzilli — COMMENTED** [review]
The documentation needs to be regenerated by executing: `go run _scripts/gen-usage-docs.go`

**Review by aarzilli — APPROVED** [review]
LGTM

**Review by derekparker — APPROVED** [review]
LGTM

**Pull request merged**